### PR TITLE
chore(z-index): tidy up z-index ordering

### PR DESCRIPTION
**Pull request by gepifanio**
Ticket: #982

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit ca66f1540a204739c4061538b2d963ca83de2e2c

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5d11b51cd5ca1b0007fe5a3a

File: components/page/megamenu/_mega-menu.css [hunk 1]
```diff
@@ -183,7 +183,7 @@
 
       &::after {
         content: '\203a';
-        z-index: 9;
+        z-index: (var(--z-default) + 8);
```
Comment (TimMansell) [inline components/page/megamenu/_mega-menu.css]: It looks like we're not using  `&__link-nested` class at all.  can remove

File: components/page/megamenu/_mega-menu.css [hunk 2]
```diff
@@ -481,7 +481,7 @@
       &__section .menu__item[tabindex='0']:focus {
         .inner {
           display: flex;
-          z-index: 3;
+          z-index: (var(--z-default) + 2);
```
Comment (TimMansell) [inline components/page/megamenu/_mega-menu.css]: don't need this

File: components/page/nav/_page-nav.css [hunk 3]
```diff
@@ -11,7 +11,7 @@ page-nav {
 
 .sitenav {
   position: fixed;
-  z-index: 101;
+  z-index: (var(--z-default) + 100);
```
Comment (TimMansell) [inline components/page/nav/_page-nav.css]: wrap in `calc()` & use --z-navigation + 1

File: components/page/nav/_page-nav.css [hunk 4]
```diff
@@ -43,7 +43,7 @@ page-nav {
 
     /* hide sidebar of parent panels */
     &--nested-open {
-      z-index: 102;
+      z-index: (var(--z-default) + 101);
```
Comment (TimMansell) [inline components/page/nav/_page-nav.css]: wrap in `calc()` & use --z-navigation + 2

File: components/page/navigation/index.css [hunk 5]
```diff
@@ -49,7 +49,7 @@
     justify-content: center;
     align-self: center;
     position: fixed;
-    z-index: 1000;
+    z-index: (var(--z-default) + 999);
```
Comment (TimMansell) [inline components/page/navigation/index.css]: wrap in `calc()` & use --z-overlay

File: components/page/nav/_page-nav.css [hunk 6]
```diff
@@ -83,7 +83,7 @@ page-nav {
   &__nested-trigger {
     &::after {
       content: '\203a';
-      z-index: 9;
+      z-index: (var(--z-default) + 8);
```
Comment (TimMansell) [inline components/page/nav/_page-nav.css]: wrap in `calc()`

File: components/page/search/_blanket.css [hunk 7]
```diff
@@ -1,7 +1,7 @@
 .blanket {
   display: none;
   position: fixed;
-  z-index: 100;
+  z-index: (var(--z-default) + 99);
```
Comment (TimMansell) [inline components/page/search/_blanket.css]: wrap in `calc()` & use --z-navigation -1

File: components/page/search/_blanket.css [hunk 8]
```diff
@@ -16,7 +16,7 @@
 
   &.white {
     display: block;
-    z-index: 100;
+    z-index: (var(--z-default) + 99);
```
Comment (TimMansell) [inline components/page/search/_blanket.css]: wrap in `calc()` & use --z-navigation -1

File: components/page/search/_header-tools.css [hunk 9]
```diff
@@ -1,6 +1,6 @@
 .header-tools {
   position: absolute;
-  z-index: 101;
+  z-index: (var(--z-default) + 100);
```
Comment (TimMansell) [inline components/page/search/_header-tools.css]: wrap in `calc()` & use --z-navigation +10

File: components/page/search/_header-tools.css [hunk 10]
```diff
@@ -56,7 +56,7 @@
 .page-header-search {
   display: none;
   position: fixed;
-  z-index: 3;
+  z-index: (var(--z-default) + 2);
```
Comment (TimMansell) [inline components/page/search/_header-tools.css]: don't need this

File: components/timeline/index.css [hunk 11]
```diff
@@ -45,7 +45,7 @@
       content: '';
       display: block;
       position: absolute;
-      z-index: 2;
+      z-index: (var(--z-default) + 1);
```
Comment (TimMansell) [inline components/timeline/index.css]: don't need this.

File: components/toggle/index.css [hunk 12]
```diff
@@ -7,7 +7,7 @@
   &__default {
     display: flex;
     position: relative;
-    z-index: 1;
+    z-index: var(--z-default);
```
Comment (TimMansell) [inline components/toggle/index.css]: don't need this.

**Pull request merged**